skietbaan_fe/src/components/CreatePage.js

```diff
@@ -30,14 +31,17 @@ class CreatePage extends Component {
 	}
 
 	ViewMembers() {
-		window.location = "/viewmembers";
+		window.location = "/viewMembers";
 	}
 
 	ViewGroups() {
 		window.location = "#";
 	}
 
 	render() {
+		if(!getCookie("token")){
+            window.location = "/register-page";
```

Comment:
Others are camelCases change this to match please

Reply:
okay